apps/web/components/shared/kyc/kyc-4/kyc-4-upload.tsx

```
@@ -85,244 +60,154 @@ const ProofOfAddressUpload = ({
+		}
+		const selectedFile = e.target.files?.[0]
+		if (selectedFile && isValidFileType(selectedFile)) {
+			await handleFileUpload(selectedFile, documentType, setFile, toast)
+		}
+	}
+	
+	const removeFile = (
+		previewUrl: string | null,
+		setFile: (file: File | null) => void,
+		setPreviewUrl: (url: string | null) => void,
+		setExtractedData: (data: any) => void,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)